README.md

```diff
@@ -480,6 +483,23 @@ skale resources-allocation generate
 Options:
 
 -   `--yes` - generate without additional confirmation
+-   `-f/--force` - rewrite allocation file if it exists
+
+### Validate commands
+
+> Prefix: `skale validate`
+
+#### Validate abi
+
+Check if abi failes present and have valid json data 
```

Comment:
"Check whether ABI and JSON data is valid"

Reply:
I think it's better to correct to "Check whether ABI files contain valid JSON data"